apps/web/src/actions/connectedEvaluations/update.ts

```diff
@@ -0,0 +1,33 @@
+'use server'
+
+import { ConnectedEvaluationsRepository } from '@latitude-data/core/repositories'
+import { updateConnectedEvaluation } from '@latitude-data/core/services/connectedEvaluations/update'
+import { z } from 'zod'
+
+import { authProcedure } from '../procedures'
+
+export const updateConnectedEvaluationAction = authProcedure
```

Comment:
This is a more `toggleLiveAction` so we can track individual events when it happens

Reply:
hmm it's a good point, will do in another pr also